packages/node_modules/@webex/plugin-meetings/src/meetings/index.js

```diff
@@ -169,6 +170,18 @@ export default class Meetings extends WebexPlugin {
 
           return;
         }
+
+
+        // When its wireless share or guest and user leaves the meeting we dont have to keep the meeting object
+        // Any future events will be neglected
+
+        if (data.locus && data.locus.self && (data.locus.self.state === _LEFT_ && data.locus.self.removed === true)) {
```

Comment:
Could remove the extra `()`?

```suggestion
        if (data.locus && data.locus.self && data.locus.self.state === _LEFT_ && data.locus.self.removed === true) {
```